README.md

```diff
@@ -303,33 +303,34 @@ have an understanding of the topic yourself, and know what questions to ask.
 
 ## Fundamentals of Stock Corporations
 
-Before getting into how equity is used for compensation, it’s important to understand some
-fundamentals about how companies are formed and owned.
-What are stock and shares?
-And how are shares in a company created and owned, and how does that change over time?
+Before getting into the ways in which equity can be used to compensate employees in a
```

Comment:
Not a big deal in this case, as this intro likely should be rewritten anyway.

Reply:
I added "in a company" here b/c I felt that it otherwise wasn't clear why we were beginning with companies and corporations. Wanted to bring the themes together. Can revert.

Reply:
Not a strong opinion on this one. With langauge like this,  I find myself wondering, "an employee of what, if not a company?"

Do agree with needing a better intro. I've added a Notion item to rewrite this intro.

Reply:
Fixed.